src/cases/miniplex/entity_cycle.js

```diff
@@ -0,0 +1,22 @@
+import { World } from "miniplex";
+
+export default (count) => {
+  const ecs = new World();
+
+  for (let i = 0; i < count; i++) {
+    ecs.createEntity({ A: 1 });
+  }
+
+  const withB = ecs.archetype("B");
+
+  return () => {
+    for (let i = ecs.entities.length; i > 0; i--) {
```

Comment:
You must iterate over entities with component `A`. I know it does not make sense since all existing entities as the component `A`, but it must be changed to be fair with the other implementations.

Reply:
I've made the change, thanks for pointing this out.